certbot-apache/certbot_apache/configurator.py

```diff
@@ -94,26 +95,49 @@ class ApacheConfigurator(augeas_configurator.AugeasConfigurator):
 
     OS_DEFAULTS = dict(
         server_root="/etc/apache2",
-        vhost_root="/etc/apache2/sites-available",
+        vhost_root="/etc/apache2/sites-enabled",
         vhost_files="*",
         logs_root="/var/log/apache2",
+        ctlpath="apache2ctl",
+        binpath="apache2",
```

Comment:
I thought that it might clarify the need of full paths for users that are running custom Apache installations (the main demographic for users of these flags), as these might not have the `$PATH` correctly set.

Reply:
I do agree, this is now changed. Shorter options are always nicer, especially if you need to define multiple of them.